Xcode-config/Shared.xcconfig

```diff
@@ -0,0 +1,38 @@
+#include "DEVELOPMENT_TEAM.xcconfig"
+
+// Create the file DEVELOPMENT_TEAM.xcconfig
+// in the "Xcode-config" directory within the project directory
+// with the following build setting:
+// DEVELOPMENT_TEAM = [Your TeamID]
+
+// Hint: recent Xcode versions appear to automatically create an empty file 
+// for you on the first build. This build will fail, or course, 
+// because code-signing can’t work without the DEVELOPMENT_TEAM set. 
+// Just fill it in and everything should work. 
+
+// You can find your team ID by logging into your Apple Developer account
+// and going to
+// https://developer.apple.com/account/#/membership
+// It should be listed under “Team ID”.
+
+// To set this system up for your own project,
+// copy the "Xcode-config" directory there,
+// add it to your Xcode project,
+// navigate to your project settings
+// (root icon in the Xcode Project Navigator)
+// click on the project icon there,
+// click on the “Info” tab
+// under “Configurations”
+// open the “Debug”, “Release”,
+// and any other build configurations you might have.
+// There you can set the pull-down menus in the
+// “Based on Configuration File” column to “Shared”.
+// Done.
+
+// Don’t forget to add the DEVELOPMENT_TEAM.xcconfig file to your .gitignore:
```

Comment:
This comment seems unnecessary; you're already adding it.

Reply:
It’s for when people want to adapt the concept and add it to their projects.

Reply:
OK, that makes sense.

Reply:
Does this need additional work? I resolved a CI issue with this approach here, if you are interested:
https://github.com/git-up/GitUp/pull/641

Reply:
i haven't had a good chance to actually test it out, but I will get to it as soon as I can.

Reply:
Thank you!

Reply:
Any comments?